examples/jsm/loaders/EXRLoader.js

```diff
@@ -1995,7 +2159,9 @@ class EXRLoader extends DataTextureLoader {
 				'B44_COMPRESSION',
 				'B44A_COMPRESSION',
 				'DWAA_COMPRESSION',
-				'DWAB_COMPRESSION'
+				'DWAB_COMPRESSION',
+				'UNKNOWN_COMPRESSION', // 10
```

Comment:
EXR compression test:
[htj2k256 float 32.exr](https://github.com/TodicaIonut/Test-EXR/blob/main/htj2k256%20float%2032.exr) test
[htj2k256 half 16.exr](https://github.com/TodicaIonut/Test-EXR/blob/main/thj2k256%20half%2016.exr) test
[htj2k32 float 32.exr](https://github.com/TodicaIonut/Test-EXR/blob/main/htj2k32f%20float%2032.exr) test
[htj2k32 half 16.exr](https://github.com/TodicaIonut/Test-EXR/blob/main/thj2k32%20half%2016.exr) test

```suggestion
				'HTJ2K256_COMPRESSION', // 10
				'HTJ2K32_COMPRESSION' // 11
```